docs/v1/P4Runtime-Spec.mdk

```diff
@@ -6269,6 +6279,356 @@ field [@ProtoAny]. At the moment, there is not any mechanism to extend the
 `p4.v1.TableEntry` message based on the value of architecture-specific table
 properties, but we may include on in future versions of the API.
 
+# GenericTable { #sec-generic-table}
+
+## GenericTable p4info { #sec-p4info-generic-table}
+
+`GenericTable` messages are used to map non-PSA P4-externs or non-P4 target-
+specific "fixed" features to their implementation in a generic way. The same can be achieved
+via `Extern`, but GenericTable provides a structured way in which every feature is
+represented as a set of match-fields and data-fields. The data consists of unions
+which are similar to actions. One use of GenericTable in a server backend is
+with TDI [Table Driven Interface](https://github.com/p4lang/tdi), but targets
+can use Generic table to map to their own specific feature implementations as
+well.
+It defines the following fields
+
+* `generic_table_type_id`, a 8-bit unsigned integer which uniquely identifies the
+  generic_table_type in the context of the architecture.
```

Comment:
I believe this is the first time that generic_table_type is mentioned in the spec.  It seems like a good place to either explain this new concept with an example, or to link to a place where such an explanation is given elsewhere in the spec.

For example, when would an architecture use the same generic_table_type for two or more different objects?  When would it use different generic_table_type's?  Why?

Reply:
Added some more text. Please check